scipy/stats/tests/test_discrete_distns.py

```diff
@@ -574,3 +575,46 @@ def test_gh_17146():
     assert_allclose(pmf[-1], p)
     assert_allclose(pmf[0], 1-p)
     assert_equal(pmf[~i], 0)
+
+
+@pytest.mark.parametrize('x, n, a, b, ref',
+                         [[5, 5e6, 5, 20, 1.1520944824139114e-107],
+                          [100, 50, 5, 20, 0.002855762954310226],
+                          [10000, 1000, 5, 20, 1.9648515726019154e-05]])
+def test_betanbinom_pmf(x, n, a, b, ref):
+    # test that PMF stays accurate in the distribution tails
+    # reference values computed with mpmath
+    # from mpmath import mp
+    # mp.dps = 500
+    # def betanbinom_pmf(k, n, a, b):
+    #     k = mp.mpf(k)
+    #     a = mp.mpf(a)
+    #     b = mp.mpf(b)
+    #     n = mp.mpf(n)
+    #     return float(mp.binomial(n + k - mp.one, k) * mp.beta(a + n, b + k)
+    #                  / mp.beta(a, b))
+    assert_allclose(betanbinom.pmf(x, n, a, b), ref, rtol=1e-10)
+
+
+@pytest.mark.parametrize('n, a, b, ref',
+                         [[10000, 5000, 50, 0.12841520515722202],
+                          [10, 9, 9, 7.9224400871459695],
+                          [100, 1000, 10, 1.5849602176622748]])
+def test_betanbinom_kurtosis(n, a, b, ref):
+    # reference values were computed via mpmath
+    # from mpmath import mp
+    # def kurtosis_betanegbinom(n, a, b):
+    #     n = mp.mpf(n)
+    #     a = mp.mpf(a)
+    #     b = mp.mpf(b)
+    #     four = mp.mpf(4.)
+    #     mean = n * b / (a - mp.one)
+    #     var = (n * b * (n + a - 1.) * (a + b - 1.)
+    #            / ((a - 2.) * (a - 1.)**2.))
+    #     def f(k):
+    #         return (mp.binomial(n + k - mp.one, k) * mp.beta(a + n, b + k)
+    #                  / mp.beta(a, b) * (k - mean)**four)
+    #         fourth_moment = mp.nsum(f, [0, mp.inf])
+    #     return float(fourth_moment/var**2 - 3.)
+    _, _, _, k = betanbinom.stats(n, a, b, moments="msvk")
```

Comment:
k = betanbinom.stats(n, a, b, moments="k")

add a blank line at the end of the file